src/main/java/com/terraformersmc/terrestria/feature/tree/trunkplacers/MegaTrunkPlacer.java

```diff
@@ -38,22 +36,22 @@ public MegaTrunkPlacer(int height, int randomHeight, int extraRandomHeight) {
 		return TrunkPlacerType.GIANT_TRUNK_PLACER;
```

Comment:
Can you incorporate the fix in #263 here

Reply:
Done!